docs/Custom_Pagers.md

```diff
@@ -26,6 +26,8 @@ git:
 
 ![](https://i.imgur.com/QJpQkF3.png)
 
+A cool feature of delta is --hyperlinks, which renders clickable links for the line numbers in the left margin, and lazygit supports these. To use them, set the `pager:` config to `delta --dark --paging=never --line-numbers --hyperlinks`. If you are a VS Code user, I recommend to add `--hyperlinks-file-link-format="vscode://file/{path}:{line}`; this allows you to click on an underlined line number in the diff to jump right to that same line in your editor.
```

Comment:
Might be worth also mentioning that JetBrains IDEs supports URLs. I believe (but have not recently tested) that examples are

```
hyperlinks-file-link-format = "idea://open?file={path}&line={line}"
hyperlinks-file-link-format = "pycharm://open?file={path}&line={line}"
```

But presumably similar exists for the other JetBrains IDEs (Go etc).

Reply:
Instead of mentioning these individually, I decided to link to your open-in-editor project, which I would hope will be kept up to date as other editors add their own URL schemes, so that we don't have to do this in two places. See afdd3b6ab802.

Reply:
Hm, sorry, I don't think that linking to open-in-editor is the right thing to do! That project is very difficult to use , doesn't support Windows, and isn't necessary for users whose editors support their own URL protocol (VSCode, JetBrains IDEs, Zed, maybe others). It would be a confusing experience for visitors since the README basically suggests not to use it unless you really have to! It's also inactive, and I don't use it myself currently (my current solution is https://github.com/dandavison/wormhole but that's definitely not ready to link to, although you might be interested in it.)

Reply:
I think the right solution here is for me to add a "hyperlinks" section to the delta manual; I'll do that and post back here.

Reply:
Done: https://dandavison.github.io/delta/hyperlinks.html

Reply:
Very nice, thanks, that's a great writeup. I like the suggestion with the small python http server to solve this.

Linked to it in 01f65ed609aa.

Reply:
Wait, this is all still too complicated. Users already have an editor configuration which is used when they press `e` in lazygit; they shouldn't have to do yet another editor-specific configuration orgy for this hyper-link feature. It should just use that config.

I added a commit (752ce3e510aa) that makes it possible to set the hyperlink format to a `lazygit-edit:` URL, and it will recognize that and call the editor command. I think that's the best solution. (Documentation update in 7d53179b7843.)

Reply:
That makes sense! (I'll have to look into what you do for `lazygit-edit`). I'm happy to have the new section for the delta manual in any case.

Reply:
If they don't use the `lazygit-edit` URL format, will lazygit pass the URL on to the operating system's standard protocol handling mechanisms (so that e.g. `http://` issues an http request)?

Reply:
Yes.